packages/run-utils/tests/main.js

```diff
@@ -0,0 +1,44 @@
+const test = require('ava')
+const { valid: validVersion } = require('semver')
+
+const run = require('..')
+
+test('Properties', t => {
+  t.is(typeof run, 'function')
+  t.is(typeof run.command, 'function')
+})
+
+test('Can run local binaries', async t => {
+  const { stdout } = await run('ava', ['--version'])
+  t.truthy(validVersion(stdout))
+})
+
+test('No arguments', async t => {
+  const { stdout } = await run('echo')
+  t.is(stdout, '')
+})
+
+test('Empty options object', async t => {
+  const { stdout } = await run('echo', {})
+  t.is(stdout, '')
+})
+
+test('Option stdio', async t => {
```

Comment:
Is there a way we can validate output is not piped to the parent?
Currently the validation for theses tests is identical to `Can run local binaries`

Reply:
You're right, those tests are not good at the moment. :+1: 
I just fixed the tests to validate this.